packages/init/src/bin.ts

```diff
@@ -11,10 +11,15 @@ const info = require('../package.json');
 
 const argv = yargs(hideBin(process.argv))
   .usage(`Usage: ${info.name} [options]`)
+  .option('custom-version', {
```

Comment:
can we change it to just `--version` or will it intercept the version of the program?

Reply:
yeah, `--version` was already taken

Reply:
so how about `--repack-version` or `--exact-version`? I'm thinking about long-term using custom and it feels ambiguous